docs/advanced-features/security-headers.md

```diff
@@ -0,0 +1,136 @@
+---
+description: Improve the security of your Next.js application by adding HTTP response headers.
+---
+
+# Security Headers
+
+To improve the security of your application, you can use [`headers`](/docs/api-reference/next.config.js/headers.md) in `next.config.js` to apply HTTP response headers to all routes in your application.
+
+```jsx
+// next.config.js
+
+// You can choose which headers to add to the list
+// after learning more below.
+const securityHeaders = [];
+
+async headers() {
+  return [
+    {
+      // Apply these headers to all routes in your application.
+      source: '/(.*)',
+      headers: securityHeaders
+    }
```

Comment:
Hello, array closing bracket is missing here.

Reply:
Nice catch, would you like to submit a PR to fix it?

Reply:
@styfle Hi sure , just submitted pull request here: #26375